plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/MetadataUtil.java

```diff
@@ -24,18 +24,18 @@
 {
     private MetadataUtil() {}
 
-    public static String getEscapedTableName(Optional<String> schema, String table)
+    public static @Nullable String getEscapedTableName(@Nullable String schema, String table)
     {
-        return SchemaUtil.getEscapedTableName(toPhoenixSchemaName(schema).orElse(null), table);
+        return SchemaUtil.getEscapedTableName(toPhoenixSchemaName(schema), table);
     }
 
-    public static Optional<String> toPhoenixSchemaName(Optional<String> trinoSchemaName)
+    public static @Nullable String toPhoenixSchemaName(@Nullable String trinoSchemaName)
     {
-        return trinoSchemaName.map(schemaName -> DEFAULT_SCHEMA.equalsIgnoreCase(schemaName) ? NULL_SCHEMA_NAME : schemaName);
+        return DEFAULT_SCHEMA.equalsIgnoreCase(trinoSchemaName) ? NULL_SCHEMA_NAME : trinoSchemaName;
     }
 
-    public static Optional<String> toTrinoSchemaName(Optional<String> phoenixSchemaName)
+    public static @Nullable String toTrinoSchemaName(@Nullable String phoenixSchemaName)
     {
-        return phoenixSchemaName.map(schemaName -> schemaName.isEmpty() ? DEFAULT_SCHEMA : schemaName);
+        return phoenixSchemaName.equalsIgnoreCase(NULL_SCHEMA_NAME) ? DEFAULT_SCHEMA : phoenixSchemaName;
```

Comment:
`phoenixSchemaName` is `@Nullable`, so calling `equalsIgnoreCase` on it may NPE.